README.md

```diff
@@ -68,30 +68,86 @@ A Python package manager written in Rust inspired by Cargo.
 Usage: huak <COMMAND>
 
 Commands:
-  activate  Activate the project's virtual environment
-  add       Add a dependency to the existing project
-  audit     Check for vulnerable dependencies and license compatibility*
-  build     Build tarball and wheel for the project
-  clean     Remove tarball and wheel from the built project
-  doc       Generates documenation for the project*
-  fix       Auto-fix fixable lint conflicts
-  fmt       Format the project's Python code
-  init      Initialize the existing project
-  install   Install the dependencies of an existing project
-  lint      Lint the project's Python code
-  new       Create a new project at <path>
-  publish   Builds and uploads current project to a registry*
-  remove    Remove a dependency from the project
-  run       Run a command within the project's environment context
-  test      Test the project's Python code
-  update    Update dependencies added to the project*
-  version   Display the version of the project
-  help      Print this message or the help of the given subcommand(s)
+  activate    Activate the project's virtual environment
+  add         Add a dependency to the existing project
+  audit       Check for vulnerable dependencies and license compatibility*
+  build       Build tarball and wheel for the project
+  completion  Generates a shell completion script for supported shells. See the help menu for more information on supported shells
+  clean       Remove tarball and wheel from the built project
+  doc         Generates documentation for the project*
+  fix         Auto-fix fixable lint conflicts
+  fmt         Format the project's Python code
+  init        Initialize the existing project
+  install     Install the dependencies of an existing project
+  lint        Lint the project's Python code
+  new         Create a new project at <path>
+  publish     Builds and uploads current project to a registry*
+  remove      Remove a dependency from the project
+  run         Run a command within the project's environment context
+  test        Test the project's Python code
+  update      Update dependencies added to the project*
+  version     Display the version of the project
+  help        Print this message or the help of the given subcommand(s)
```

Comment:
How come this entire diff is here? It looks okay, but I just thought that was strange.

Reply:
I actually also found that weird at first but the `completion` command is the new longest command so all the other commands get new tab spacing.

Reply:
ah. makes sense. thanks for pointing that out.